config.go

```diff
@@ -20,19 +23,23 @@ type TargetConfig struct {
 	Target string
 	// Timeout is the timeout to use for this specific target if it is different to DefaultTimeout
 	Timeout time.Duration
+	// HTTPClientTimeout is the timeout for requests made by a http client
+	HTTPClientTimeout time.Duration `yaml:"http_client_timeout"`
 }
 
 // Config represents all of the config that can be defined in a config file
 type Config struct {
-	DefaultTimeout time.Duration `yaml:"default-timeout"`
-	Targets        map[string]TargetConfig
+	DefaultTimeout           time.Duration `yaml:"default-timeout"`
+	Targets                  map[string]TargetConfig
+	DefaultHTTPClientTimeout time.Duration `yaml:"default_http_client_timeout"`
```

Comment:
The convention I've been using for YAML fields is kebab-case. Could you use the same for this please.